dm/master/server.go

```diff
@@ -1994,3 +2003,12 @@ func (s *Server) HandleHTTPApis(ctx context.Context, mux *http.ServeMux) error {
 
 	return nil
 }
+
+func (s *Server) serveHostAndPort() (host, port string, err error) {
```

Comment:
any better name rather than `serve`? and compared with `net.SplitHostPort(s.cfg.MasterAddr)` directly, only wrapped the `err` with `ErrMasterHostPortNotValid`?

Reply:
yes, only wrap an internal error, change the function name to `splitHostPort`